gruntfile.js

```diff
@@ -13,6 +13,23 @@ module.exports = function (grunt) {
 
     require('time-grunt')(grunt);
 
+    var concatConfig = {
+        target: {
+            src: [
+                'src/**/*.js'
+            ],
+            dest: 'dist/angular-token-auth.js'
+        }
+    };
+
+    var uglifyConfig = {
+        target: {
+            files: {
+                'dist/angular-token-auth-mini.js': 'dist/angular-token-auth-mini.js'
```

Comment:
This line is minifying a file to itself. I think you must have meant for the right side to be `dist/angular-token-auth.js`?